common/lib/xmodule/xmodule/raw_module.py

```diff
@@ -4,8 +4,8 @@
 
 from lxml import etree
 from xblock.fields import Scope, String
-from xmodule.editing_module import XMLEditingDescriptor
-from xmodule.xml_module import XmlDescriptor
+from xmodule.editing_module import XMLEditingDescriptor  # pylint: disable=unused-import
+from xmodule.xml_module import XmlDescriptor  # pylint: disable=unused-import
```

Comment:
For all of these `disable=unused-import`, I think it would be helpful to have a short comment explaining why they're still needed.

Reply:
Since it was just a refactoring PR, I didn't remove any import which could've been in use anywhere else and these will be removed later on after proper testing. 
These and all other instances, don't have any details because most of these have been applied automatically using lint-amnesty before this.